src/server/services/pickup/hooks.js

```diff
@@ -1,19 +1,20 @@
 import configureServer from './configure-server';
-import createMumbleChannel from './create-mumble-channel';
-import removeMumbleChannel from './remove-mumble-channel';
 
 export default {
   after: {
     create(props) {
-      createMumbleChannel(props);
       configureServer(props);
 
+      props.app.service('mumble').create(props.status.region, props.status.id);
```

Comment:
props.status.region is not a thing, I guess you mean props.result.region?

Reply:
fml, Idk why I typed status